File: src/main/java/nl/dannyjelsma/soundboardbot/listeners/NewGuildListener.java
package nl.dannyjelsma.soundboardbot.listeners;

import net.dv8tion.jda.api.events.guild.GuildJoinEvent;
import net.dv8tion.jda.api.hooks.ListenerAdapter;
import nl.dannyjelsma.soundboardbot.audio.AudioPlayerSendHandler;
import nl.dannyjelsma.soundboardbot.audio.SoundManager;

public class NewGuildListener extends ListenerAdapter {

    @Override
    public void onGuildJoin(GuildJoinEvent event) {
        event.getGuild().getAudioManager().setSendingHandler(new AudioPlayerSendHandler(SoundManager.getInstance().getAudioPlayer()));
    }
}

File: src/main/java/nl/dannyjelsma/soundboardbot/listeners/NewGuildListener.kt
package nl.dannyjelsma.soundboardbot.listeners

import net.dv8tion.jda.api.events.guild.GuildJoinEvent
import net.dv8tion.jda.api.hooks.ListenerAdapter
import nl.dannyjelsma.soundboardbot.audio.AudioPlayerSendHandler
import nl.dannyjelsma.soundboardbot.audio.SoundManager

class NewGuildListener : ListenerAdapter() {
    override fun onGuildJoin(event: GuildJoinEvent) {
        event.guild.audioManager.sendingHandler = AudioPlayerSendHandler(SoundManager.audioPlayer)
    }
}